torch/nn/functional.py

```diff
@@ -2299,6 +2299,30 @@ def pairwise_distance(x1, x2, p=2, eps=1e-6, keepdim=False):
     return torch.pairwise_distance(x1, x2, p, eps, keepdim)
 
 
+pdist = _add_docstr(torch.pdist, r"""
+pdist(input, p=2) -> Tensor
+
+Computes the p-norm distance between every pair of row vectors in the input.
+This is identical to the upper triangular portion, excluding the diagonal, of
+`torch.norm(input[:, None] - input, dim=2, p=p)`. This function will be faster
+if the rows are contiguous.
+
+If input has shape :math:`N \times M` then the output will have shape
+:math:`\frac{1}{2} N (N - 1)`.
```

Comment:
You are missing a \times M here.

Reply:
Where?

Reply:
Oh my bad. I thought it returns a 2d tensor. Sorry about it!

Reply:
Might make it clearer if you print it as a tuple (e.g. (n/(n-1),)

Reply:
I was generally confused when reading documentation. Is math or cs notation preferred? I.e. should I replace all shapes with tuples e.g. `(n * (n - 1) // 2,)`?